packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Arrange from 'terra-arrange';
+import ResponsiveElement from 'terra-responsive-element';
+import NavHelpContent from './NavHelpContent';
+import IconInfo from '../../icons/IconInfo';
+import Modal from '../modal/Modal';
+import Popup from '../popup/Popup';
+import navElementShape from '../../NavPropShapes';
+import styles from './NavHelp.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * An array of items to be displayed as help menu/popup.
+   */
+  help: PropTypes.arrayOf(PropTypes.shape(
+    navElementShape,
+    {
+      children: PropTypes.arrayOf(PropTypes.shape(
+      navElementShape,
+        {
+          children: PropTypes.array,
+        },
+      )),
+    },
+  )),
+  /**
+   * A unique id set to the help button that will be referred in help menu/popup .
+   */
+  id: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+  help: [],
+};
+
+const translations = {
+  help: 'Help',
+};
+
+class NavHelp extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+    this.togglePopup = this.togglePopup.bind(this);
+  }
+
+  togglePopup() {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  render() {
+    const { help, id, ...customProps } = this.props;
+    const helpButton = (
+      <button
+        id={id}
+        onClick={this.togglePopup}
+        className={cx('nav-help')}
+      >
+        <Arrange
+          fill={<div className={cx('icon')}><IconInfo /></div>}
+          fitEnd={<div className={cx('button-text-padding')}>{translations.help}</div>}
+          align="stretch"
+        />
+      </button>);
+
+    const popupContent = <NavHelpContent helpContent={help} />;
+
+    const defaultElement = (<Modal
+      isModalOpen={this.state.isOpen}
+      title={translations.help}
+      content={popupContent}
+      closeModal={this.togglePopup}
+    />);
+
+    const popup = (<Popup
+      title={translations.help}
+      hasHeader
+      isOpen={this.state.isOpen}
+      targetRef={() => document.getElementById(id)}
+      closePopup={this.togglePopup}
+      contentWidth="320"
+      contentHeight="240"
+      classNameContent={cx('popup-content')}
+      contentAttachment="middle center"
+      popupContent={popupContent}
+    />);
+
+    return (
+      <div {...customProps}>
+        <ResponsiveElement responsiveTo="window" defaultElement={defaultElement} medium={popup} large={popup} />
```

Comment:
you don't need to set the same content for medium and large, just set medium.

> It is not necessary to set each breakpoint. If a breakpoint is not set the component will look at each successive smaller breakpoint until one is found.
**http://terra-ui.herokuapp.com/components/core/responsive-element**